scripts/configure_postgres.sh

```diff
@@ -4,7 +4,12 @@
 set -e -x
 
 if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
-    CONFIG_DIR=/etc/postgresql/9.6/main/
+    # ARM64 is supported from postgresql version 10
```

Comment:
Could you please make a test whether we can move to postgres 10 to all the platforms? This 9.6 has been probably here since it was the new kid on the block.

Reply:
Updated as requested.